org.civicrm.reqangular/src/common/services/notification.js

```diff
@@ -1,56 +1,23 @@
+/* globals CRM */
```

Comment:
still as per the angular style guide, if the name of the service is a noun, you should append "Service" to it. So it should be "notificationService" i think